MediaBrowser.Plugins.Anime/Configuration/configPage.html

```diff
@@ -36,13 +37,25 @@
                             <label for="chkMaxGenres">
                                 Max Genres [0: unlimited]
                             </label>
-                            <input id="chkMaxGenres" name="chkMaxGenres" type="number" min="0" value="maxGenres"/>
+                            <input id="chkMaxGenres" name="chkMaxGenres" type="number" min="0" value="maxGenres" />
+                        </li>
+                        <li>
+                            <label for="chkMyAnimeList_API_Name">
+                                MyAnimeList Name. -not rly needed
+                            </label>
+                            <input id="chkMyAnimeList_API_Name" name="chkMyAnimeList_API_Name" type="text" value="" disabled/>
```

Comment:
it really needed when accessing API  (search in this case)

Reply:
not rly there are some other ways and i dont want too save the password and username in clear text but you can unlock it for yourself it will speed up your libary scan because it dont need too use th webpage

Reply:
@DaniGTA i'm saying this according to the code exists, one part parsing web page - which is not require user name/pass; second doing API search - what request permission; Moreover ..if spam a lot o f search via API without authorization, it counts as auth fail and bans from auth for some time

Reply:
oh i see it doesnt have a fallback i fix this now and it didnt prevent the WebRequest if no username is entered.